internal/app/test-fixtures/test_config.yaml

```diff
@@ -25,4 +25,7 @@ policies:
   - local: /tmp
   - repo: https://bla.com
 outputs:
-  - file: /some/file.json
\ No newline at end of file
+  file: /some/file.json
+  pubsub:
+  project: my-pubsub-project
```

Comment:
Missing indent, `project` and `topic` should be "under" `pubsub`

Reply:
Fixed